runtime/src/bank.rs

```diff
@@ -6534,14 +6544,58 @@ impl Bank {
         new_feature_activations: &HashSet<Pubkey>,
     ) {
         for builtin in BUILTINS.iter() {
+            // The `builtin_is_bpf` flag is used to handle the case where a
```

Comment:
Does the SIMD specify the behavior in this case?

Reply:
It does not. In general it doesn't really make sense to add a feature gate for activating a builtin and also using that same feature gate to migrate it to BPF. One would just deploy the program directly to BPF.

Reply:
Then why add logic for it?

Reply:
@Lichtso based on below discussions, I think this makes sense to keep. Wdyt?